bundles/org.openhab.binding.wemo/README.md

```diff
@@ -52,6 +76,7 @@ Devices support some of the following channels:
 | timespan            | Number        | Time in seconds over which onTotal applies. Typically 2 weeks except first used.                                           | Insight                                              |
 | averagePower        | Number:Power  | Average power consumption in Watts.                                                                                        | Insight                                              |
 | currentPower        | Number:Power  | Current power consumption of an Insight device. 0 if switched off.                                                         | Insight                                              |
+| currentPowerRaw     | Number:Power  | Current power consumption of an Insight device with full precision (5 mW accuracy, three decimals). 0 if switched off.     | Insight                                              |
 | energyToday         | Number:Energy | Energy in Wh used today.                                                                                                   | Insight                                              |
```

Comment:
You are introducing a breaking change. I would rather suggest to add e new channel with the reduced precision.

Reply:
> You are introducing a breaking change. I would rather suggest to add e new channel with the reduced precision.

Actually, I'm not. :-) Previously the accuracy was 1 W, i.e. rounding to nearest integer, and it still is. I tried to balance the default configuration parameters to not optimize too aggressively, so most users shouldn't notice anything, except fewer updates.

To be fully backwards compatible in terms of behavior, the sliding window could be defaulted to 0 to disable it. However, in my opinion it makes more sense to fix the problem with sensible default parameters (one minute window and 1 W forced update) and leave it to the users to adjust. I don't think any users would prefer the old behavior.

Reply:
Just to be clear: The new **currentPowerRaw** channel with the full provided precision of 5 mW accuracy is not the same as the previous **currentPower** - it never had any decimals.

Reply:
OK, convinced